packages/compass-e2e-tests/helpers/insert-data.ts

```diff
@@ -1,168 +1,217 @@
 import { MongoClient } from 'mongodb';
 import type { Db, MongoServerError } from 'mongodb';
 
-const CONNECTION_URI = 'mongodb://127.0.0.1:27091';
-
-export async function dropDatabase(db: Db | string) {
-  const database = typeof db === 'string' ? client.db(db) : db;
-  try {
-    await database.dropDatabase();
-  } catch (err) {
-    const codeName = (err as MongoServerError).codeName;
-    if (codeName !== 'NamespaceNotFound') {
-      throw err;
-    }
-  }
+import {
+  DEFAULT_CONNECTION_STRING_1,
+  DEFAULT_CONNECTION_STRING_2,
+  TEST_MULTIPLE_CONNECTIONS,
+} from './compass';
+
+// This is a list of all the known database names that get created by tests so
+// that we can know what to drop when we clean up before every test. If a new
+// test starts to create another database, add it here so that it will also be
+// cleaned up.
```

Comment:
I just know I will forget to do that 😅 Can't we get it with listDatabases?

Reply:
Sure. File a ticket :)

Reply:
(It worked this way before and I'm hesitant to do that kind of thing as a drive-by because at minimum it is going to have to filter out some system/builtin databases)